proposals/wg-auth.md

```diff
@@ -0,0 +1,87 @@
+
+## Auth OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the Auth Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”).
+The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter.
+Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+Authentication and authorization are a key requirement for registries to control access.
+Implementations of this in registries and clients have roughly followed standards set by Docker and have mostly focused on compatibility with specific implementations of registries or clients.
+This working group will define a standard to be supported by OCI compatible registries and clients when support for authentication and authorization is required.
+
+## Scope
+
+* Define registry responses to unauthenticated requests.
+* Define supported authentication methods (e.g. basic and bearer authentication).
+* Specify how clients negotiate access a repository and different types of access to that repository (e.g. pull and push).
+* Specify how clients negotiate access to multiple repositories for actions like a cross-repository blob mount.
+* Specify how clients and registries should renegotiate access for a request with expired or insufficient authorization.
+* Specify expected lifetime of registry credentials.
+* Avoid specifications that would prevent future extensibility (e.g. fine grain access control).
+* The authentication methods defined as supported should include OSS solution(s) that avoid picking/choosing a limited set of specific authentication providers as the default / winner.
+* Specify how clients and registries should be extensible/pluggable with respect to supported authentication methods.
+
+## Out of Scope
+
+* Registries and clients that do not require authentication and authorization support should be unaffected by these changes.
+* How clients store and access credentials, including credential helpers, will remain undefined.
+* Significant new functionality, not implemented by existing registries and clients, should not be created.
+
+## Intended work product
+
+* The result of this WG should be a PR to the distribution-spec.
+* Effort should be made to avoid breaking changes to existing registries and clients.
+* Effort should be made to utilize existing IETF standards when appropriate.
+
+## Proposed Owners
+
+The following have agreed to participate in the working group, review progress, report progress back to the OCI community, and present the results to the TOB once the working group has completed its objectives.
+
+* Brandon Mitchell (@sudo-bmitch)
```

Comment:
Would you mind adding me:
Toddy Mladenov (@toddysm)

Reply:
@rchincha Would you like to be part of the WG? I saw that you added zot below.

Reply:
Yes, pls add me if no concerns/objections.